internal/bminventory/inventory_test.go

```diff
@@ -5665,6 +5665,32 @@ var _ = Describe("cluster", func() {
 			})
 		})
 
+		It("Should store and retrieve the UI specific settings for a cluster when requested to do so", func() {
```

Comment:
Can you add an error case for 404 at least since we're specifically addressing that?

Reply:
Added